5-infrastructure/modules/base-env/README.md

```diff
@@ -0,0 +1,75 @@
+# Base Env Module
+This module creates the resources needed for the BOA Deloyment.
+
+The module provisions the following resources
+- 3 GKE Clusters
+    - Cluster1 in Primary Region
+    - CLuster2 in Secondary Region
+    - MCI Cluster is Primary Region
+- Bastion Host VM in the Primary Region
+- 2 Postgres CLoudSQL instances in primary and secondary region respectively
+- Shared VPC Peering for CloudSQL Instances
+- 2 Secrets to store CloudSQL Admin Passwords
```

Comment:
Isn't it the same password across both

Reply:
it is, currently in the sql module so is called twice, I can configure it to use the same secret, leaving intact the ability for user to still create as many secrets as passwords

Reply:
will open new PR for this as it will require a small change to SQL module, for now it simply creates one for each

Reply:
sq